scripts/maint/practracker/exceptions.txt

```diff
@@ -30,7 +30,7 @@
 # Remember: It is better to fix the problem than to add a new exception!
```

Comment:
Commit comment:
Is there a ticket for handling authorities and fallbacks?

Reply:
I've created #31502 for this